sonaranalyzer-dotnet/SonarAnalyzer.sln

```diff
@@ -14,22 +12,24 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Common Solution Items", "Co
 		build\Version.props = build\Version.props
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{B7233F78-E142-4882-B084-7C83BE472109}"
```

Comment:
Wow, that was super easy!

Please revert project reordering. That would make the commit very short & sweet.